ansible/configs/config_http.json

```diff
@@ -0,0 +1,8 @@
+{
+    "HttpServer": {
+      "ListenIp": "{{ ansible_host }}",
```

Comment:
I don't see ansible_host defined anywhere. However, this should be 0.0.0.0

Reply:
"ansible_host" is a magic variable: https://docs.ansible.com/ansible/latest/playbook_guide/playbooks_vars_facts.html#information-about-ansible-magic-variables

It's just the SSH IP address that Ansible is using for that particular host. 0.0.0.0 should work fine too but I wasn't sure, some apps are different but that should bind it to any available IPv4 address, so I can make that change.